win32ss/user/ntuser/menu.c

```diff
@@ -6183,29 +6175,29 @@ NtUserHiliteMenuItem(
 {
    PMENU Menu;
    PWND Window;
-   DECLARE_RETURN(BOOLEAN);
+   BOOL Ret = FALSE;
```

Comment:
your reason for changing the type was...

Reply:
The correct return type of this function is BOOL.

Reply:
Correct. Resolved and approved.